gobblin-cluster/src/main/java/org/apache/gobblin/cluster/SingleHelixTask.java

```diff
@@ -61,8 +61,13 @@ public TaskResult run() {
               this.jobName, this.jobId));
       int exitCode = this.taskProcess.waitFor();
       if (exitCode == 0) {
+        logger.info(String
```

Comment:
Instead of format, you can use {} like logger.info("Task process finished. job name: {}. job id: {}", this.jobName, this.jobId). This is more efficient since it defers the formatting.